WebContent/js/script.js

```diff
@@ -26,12 +26,22 @@ $(function () {
 
     $(".translate-trigger").click(function(e) {
     	e.preventDefault();
-
-    	updateTranslate($(this).data("lang"));
+    	
+    	if($(this).attr("data-lang") == "en"){

```

Comment:
Please, create local variable in this function: var $this = $(this);  in order to reduce the load.